### ISSUE TITLE
Support for older Android WebView

### ISSUE THREAD

**Issue by bug-1358** [anvaka/panzoom#254]
There is a compatibility problem with Android WebView (I have check for Android Web Browser for Android 4 for sure):

- function `makeDirty` from index.js use `window.requestAnimationFrame` function which need polyfill
- external _amator_ lib use `Set()` which need polyfill
- function `applyTransform` from domController.js use css `transform: matrix()` function which is unsupported by older Android WebViews version ([MDN WebDocs](https://developer.mozilla.org/en-US/docs/Web/CSS/transform#browser_compatibility)

First two glitches can be fixed by adding necessary polyfills (I can recomended Polyfill.io service), but the last one require some changes in source code of panzoom:

```
function applyTransform(transform) {
    // TODO: Should we cache this?
    domElement.style.transformOrigin = '0 0 0';
    domElement.style.transform = 'matrix(' +
      transform.scale + ', 0, 0, ' +
      transform.scale + ', ' +
      transform.x + ', ' + transform.y + ')';
	
      // NOTE: older Android WebView needs -webkit- prefix for transform function
      domElement.style['-webkit-transform'] = 'matrix(' +
      transform.scale + ', 0, 0, ' +
      transform.scale + ', ' +
      transform.x + ', ' + transform.y + ')';
  }
```

In short: my proposition is adding `-webkit-transform: matrix()` to transform function and everything should works.
